src/map/battle.cpp

```diff
@@ -5995,6 +5997,8 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			break;
 		case ABC_UNLUCKY_RUSH:
 			skillratio += -100 + 100 + 300 * skill_lv + 5 * sstatus->pow;
+			if (sc && sc->getSCE(SC_CHASING))
+				skillratio += 2500;
```

Comment:
```suggestion
				skillratio += 2500 * skill_lv;
```
skill description suggests "+(Lucky Rush Lv x2500)%"